frontend/src/app/libs/admin-role-management/components/users-role-management-panel/users-role-management-panel.component.ts

```diff
@@ -8,57 +8,150 @@ import { AuthFacadeService } from 'src/app/libs/auth/store/auth/auth.facade';
 import { EntityStatus } from 'src/app/libs/auth/models/entity-status';
 import { takeUntil } from 'rxjs/operators';
 import { UserSearchService } from '../../services/user-search.service';
+import { ColumnConfig, TableColumnType } from 'src/app/libs/common-components/components/table/models/row-config.enum';
+import { ComponentTheme } from 'src/app/libs/common-components/shared/component-theme.enum';
+import { RowToggleOutput } from 'src/app/libs/common-components/components/table/models/row-toggle-output.enum';
+import { TableComponent } from 'src/app/libs/common-components/components/table/table.component';
 
 @Component({
 	selector: 'app-users-role-management-panel',
 	templateUrl: './users-role-management-panel.component.html',
 	styleUrls: ['./users-role-management-panel.component.scss'],
 })
 export class UsersRoleManagementPanelComponent implements OnInit, OnDestroy {
-	public searchText: string = '';
-	public userTemplatesChanges$: Observable<SimpleChanges[]>;
+
 	public destroy$: Subject<boolean> = new Subject();
+
 	public userUpdate: EntityWrapper<User>;
+
 	public users: User[];
 
+	public isSelectedUsersRemoving: boolean;
+
+	public pendingStatus: string = EntityStatus.Pending;
+
+	public headerButtonText: string = 'delete';
+
+	public lightTheme: ComponentTheme = ComponentTheme.Light;
+	public darkTheme: ComponentTheme = ComponentTheme.Dark;
+
+	public usersRequestStatus: string;
+
+	public selectedUsers: User[];
+
+	@ViewChild(TableComponent)
+	public tableComponent: TableComponent <User>;
+
+	public columnsConfig: ColumnConfig[] = [
+		{
+			columnType: TableColumnType.Checkbox,
+			columnName: 'check',
+			dataName: 'checked'
+		},
+		{
+			columnType: TableColumnType.TextSortable,
+			columnName: 'Name',
+			dataName: 'firstName'
+		},
+		{
+			columnType: TableColumnType.Text,
+			columnName: 'Email',
+			dataName: 'email'
+		},
+		{
+			columnType: TableColumnType.Toggle,
+			columnName: 'Event Manager',
+			dataName: 'isEventManager'
+		}
+	];
+
 	constructor(
-		private userSearchService: UserSearchService,
 		private usersFacadeService: UsersFacadeService,
-		private authFacadeService: AuthFacadeService
+		private authFacadeService: AuthFacadeService,
```

Comment:
I'm not sure that comma placed at the last line of arguments is a good practice.
What do you think about that?